recipes/openlineage-sql/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "openlineage-sql" %}
+{% set version = "0.26.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/openlineage_sql-{{ version }}.tar.gz
+  sha256: 8e78db662edf1efe94da182ecfc98dbdc6ccf93a0489703c5d8a49d551760e67
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  error_overlinking: false
```

Comment:
I'm not a Rust reviewer but why is this needed here? Usually it hides over-linking problems that should be addressed. Use it with caution.

Reply:
Thanks @ocefpaf I will reach out to rust help - I dont have much experience with rust packages, and I didnt get any overlinking issues locally - not sure what I am missing here.